conans/client/graph/graph_binaries.py

```diff
@@ -145,7 +145,7 @@ def _find_existing_compatible_binaries(self, node, compatibles, remotes, update)
         conanfile = node.conanfile
         original_binary = node.binary
         original_package_id = node.package_id
-
+        node.conanfile.output.info(f"Main binary package '{original_package_id}' missing")
```

Comment:
```suggestion
        conanfile.output.info(f"Main binary package '{original_package_id}' missing")
```
Extremely minor nitpick